services/onboarding-service/src/main/java/com/amazon/aws/partners/saasfactory/saasboost/OnboardingRequest.java

```diff
@@ -35,18 +35,20 @@ public OnboardingRequest(String name) {
     }
 
     public OnboardingRequest(String name, String tier) {
-        this(name, tier, null);
+        this(name, tier, null, null);
     }
 
     @JsonCreator
     public OnboardingRequest(@JsonProperty("name") String name, @JsonProperty("tier") String tier,
-                             @JsonProperty("subdomain") String subdomain) {
+                             @JsonProperty("subdomain") String subdomain,
+                             @JsonProperty("billingPlan") String billingPlan) {
```

Comment:
The client has been sending BillingPlan the whole time as part of the onboarding requests. Has it just been dropped until now?